movement/move_accessor.py

```diff
@@ -249,20 +253,42 @@ def validate(self) -> None:
             missing_vars = set(self.var_names) - set(self._obj.data_vars)
             if missing_dims:
                 raise ValueError(
-                    f"Missing required dimensions: {missing_dims}"
+                    f"Missing required dimensions: {sorted(missing_dims)}"
```

Comment:
thanks, I missed removing that!

While figuring this out I had several `missing_dims` when using the invalid  bboxes dataset fixture (because it didn't have `keypoints`). So I needed to sort them to make the error message reproducible. But in the final solution that is not needed.

Reply:
@lochhh What do you think if we leave it in though? 🤔  

Note that we test the error raised and the [error message](https://github.com/neuroinformatics-unit/movement/blob/dc8d78e7ed083b573ddaecb3594f66f86f2af146/tests/test_unit/test_move_accessor.py#L100).

If in the future we extend our tests to fixtures with several missing dimensions/variables, then we would need the error message to be deterministic. It is kind of an annoying one to catch, because sometimes the test will pass and sometimes it won't.

But keeping it is slightly overkill atm so not sure...

Reply:
IIRC this issue arises because there was some "whole-string" comparison? Would it make more sense to just check that the error string contains the expected missing dimension(s)?